docs/source/install/rhel8.rst

```diff
@@ -0,0 +1,287 @@
+
+Install |st2| Components
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+
+  sudo yum install -y st2
+
+.. include:: common/configure_components.rst
+
+Setup Datastore Encryption
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: common/datastore_crypto_key.rst
+
+Configure SSH and SUDO
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: common/configure_ssh_and_sudo.rst
+
+Start Services
+~~~~~~~~~~~~~~
+
+.. include:: common/start_services.rst
+
+Verify
+~~~~~~
+
+.. include:: common/verify.rst
+
+Configure Authentication
+------------------------
+
+The reference deployment uses a file-based authentication provider for simplicity. Refer to
+:doc:`/authentication` to configure and use PAM or LDAP authentication backends.
+
+To set up authentication with file-based provider:
+
+* Create a user with a password:
+
+  .. code-block:: bash
+
+    # Install htpasswd utility if you don't have it
+    sudo yum -y install httpd-tools
+    # Create a user record in a password file.
+    echo 'Ch@ngeMe' | sudo htpasswd -i /etc/st2/htpasswd st2admin
+
+.. include:: common/configure_authentication.rst
+
+Install WebUI and Setup SSL Termination
+---------------------------------------
+
+`NGINX <http://nginx.org/>`_ is used to serve WebUI static files, redirect HTTP to HTTPS, provide
+SSL termination, and reverse-proxy st2auth and st2api API endpoints. To set it up: install the
+``st2web`` and ``nginx`` packages, generate certificates or place your existing certificates under
+``/etc/ssl/st2``, and configure nginx with |st2|'s supplied :github_st2:`site config file st2.conf
+<conf/nginx/st2.conf>`.
+
+|st2| depends on Nginx version >=1.7.5. RHEL has an older version in the package repositories, so
+you will need to add the official Nginx repository:
+
+.. code-block:: bash
+
+  # Add key and repo for the latest stable nginx
+  sudo rpm --import http://nginx.org/keys/nginx_signing.key
+  sudo sh -c "cat <<EOT > /etc/yum.repos.d/nginx.repo
+  [nginx]
+  name=nginx repo
+  baseurl=http://nginx.org/packages/rhel/\\\$releasever/x86_64/
+  gpgcheck=1
+  enabled=1
+  EOT"
+
+  # Ensure that EPEL repo is not used for nginx
+  sudo sed -i 's/^\(enabled=1\)$/exclude=nginx\n\1/g' /etc/yum.repos.d/epel.repo
+
+  # Install nginx
+  sudo yum install -y nginx
+
+  # Install st2web
+  sudo yum install -y st2web
+
+  # Generate a self-signed certificate or place your existing certificate under /etc/ssl/st2
+  sudo mkdir -p /etc/ssl/st2
+  sudo openssl req -x509 -newkey rsa:2048 -keyout /etc/ssl/st2/st2.key -out /etc/ssl/st2/st2.crt \
+  -days 365 -nodes -subj "/C=US/ST=California/L=Palo Alto/O=StackStorm/OU=Information \
+  Technology/CN=$(hostname)"
+
+  # Copy and enable the supplied nginx config file
+  sudo cp /usr/share/doc/st2/conf/nginx/st2.conf /etc/nginx/conf.d/
+
+  # Disable default_server configuration in existing /etc/nginx/nginx.conf
+  sudo sed -i 's/default_server//g' /etc/nginx/nginx.conf
```

Comment:
Is this right? Didn't you have to do some magic for nginx default site?

Reply:
yes, updated inline coment